civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5486,6 +5488,50 @@ function civihr_employee_portal_form_views_exposed_form_alter(&$form, &$form_sta
             }
         }
     }
+
+    if (substr($form['#id'], 0, 33) === 'views-exposed-form-civihr-report-') {
```

Comment:
Without knowing all the details on this context, I'd use something more generic like: `if( strpos($form['#id'], 'views-exposed-form-civihr-report-') !== false)` .